### ISSUE TITLE
Dev: Navigation for domain management pages

### ISSUE THREAD

**Issue by vickyszuchin** [cisagov/manage.get.gov#1136]
### Issue Description

As a domain manager
I want to understand how to navigate inside of my domain's management pages
so that I can quickly and easily get to where I want to go

The implemented "back-to" button behaviors are not consistent across the application. This scope of this task is:

- Define guidance for "back-to" button behavior for application
- Refactor "back-to" button behaviors in application to conform to approach 

## Acceptance Criteria ##
- [ ] Implement design from #1153 

### Additional Context (optional)

Suggested approaches:

1. Use breadcrumb
2. Default to one level up

### Issue Links

Blocked by #1153

**Comment by vickyszuchin** [comment]
Moving this ticket back from Planned (for sprint 41) to Backlog. This is depended on a design ticket that is currently in the backlog, #1153

**Comment by vickyszuchin** [comment]
"Blocked" tag add: to note that this ticket is currently blocked  from being story pointed by another ticket in the backlog.

**Comment by Katherine-Osos** [comment]
@PaulKuykendall @katypies Closing this ticket because its design counterpart was closed as "not planned." https://github.com/cisagov/manage.get.gov/issues/1153
